tests/tests_geomstats/test_full_rank_correlation_matrices.py

```diff
@@ -0,0 +1,144 @@
+"""Unit tests for the manifold of matrices."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.full_rank_correlation_matrices import \
+    CorrelationMatricesBundle, FullRankCorrelationAffineQuotientMetric
+from geomstats.geometry.general_linear import GeneralLinear
+from geomstats.geometry.matrices import Matrices
+from geomstats.geometry.spd_matrices import SymmetricMatrices
+
+
+class TestCorrelationMatrices(geomstats.tests.TestCase):
```

Comment:
-> `TestFullRankCorrelationMatrices`: the name of the test class is meant to match the name of the test file.